neo/Network/P2P/Payloads/WitnessScope.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Neo.Network.P2P.Payloads
+{
+    public enum WitnessScope : byte
+    {
+        /// <summary>
+        /// (neo2) - no params
+        /// </summary>
+        Global = 0x00,
+
+        /// <summary>
+        /// EntryScriptHash - no params (root-only witnesses) - can be default safe choice for native NEO/GAS (neo2 attach mode)
+        /// </summary>
+        InitScriptHash = 0x01,
```

Comment:
This should ensure that `EntryScriptHash == CallingScriptHash`, during CheckWitness verification.